view/IsoElements.py

```diff
@@ -0,0 +1,139 @@
+from abc import ABC, abstractmethod
+from typing import Tuple
+from util.Transforms import Transformations
+import pygame
+
+"""
+Bei diesr Datei handelt es sich nur um eine temporäre Datei, welche die elemtaren isometrischen Module enthält
+Später werden diese in ihre eigenen Module eingepflegt
+"""
+
+
+class Drawable(ABC):
+
+    @abstractmethod
+    def draw_block(self, window: pygame.display) -> None:
+        pass
+
+    @abstractmethod
+    def hovering(self, focus: bool = False) -> None:
+        pass
+
+    # todo methode kann evtl. auch in der Base Klasse implementiert werden, sollte eigentlich für alle gleich sein
+    @abstractmethod
+    def __nearness__(self) -> int:
+        pass
+
+
+class Block(Drawable):
+    def __init__(self, window: pygame.display, x: int, y: int, z: int) -> None:
+        self.window = window
+
+        self.block = pygame.image.load("assets/GameView/block.png")
+        self.block = pygame.transform.scale(self.block, (64, 64))
+
+        self.hovered_image = self.block.copy()
+        self.hovered_image.fill((0, 0, 0, 255), None, pygame.BLEND_RGBA_MULT)
+        self.hovered_image.fill((0, 255, 0, 0), None, pygame.BLEND_RGBA_ADD)
+
+        self.current_image = self.block
+
+        self.x = x
+        self.y = y
+        self.z = z
+
+        self.__TILEWIDTH = 64
+        self.__TILEHEIGHT = 64
+        self.nearness = 0
+
+    def draw_block(self, window: pygame.display) -> None:
+        self.nearness = self.__nearness__()
+        m_x = window.get_rect().centerx
+        m_y = window.get_rect().centery
+        v_x = self.x * self.__TILEWIDTH / 2 - self.y * self.__TILEHEIGHT / 2 + m_x
+        v_y = self.x * self.__TILEWIDTH / 4 + self.y * self.__TILEHEIGHT / 4 - self.z * self.__TILEHEIGHT / 2 + m_y / 2
+
+        self.window.blit(self.current_image, (v_x, v_y))
```

Comment:
Ich gehe mal schwer davon aus, dass sich hier die Kamera auch bereits implizit in der Rotation 45° gegen den Uhrzeigersinn befindet.

Generell ist das wiederum kein Problem wenn wir sowieso nur diese Rotation haben. Sollte aber meine Meinung nach ebenfalls ein Kommentar hin, dass die Kamera sich implizit in dieser Rotation befindet.